app/components/SignupForm/SignupForm.js

```diff
@@ -279,6 +279,7 @@ class SignupForm extends BaseForm {
                       id="name"
                       htmlFor="name"
                       label="Full name"
+                      ariaDescribedBy="Full Name"
```

Comment:
doesn't aria-describedby point to the ID of a <label>? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute